src/core/tests/application/user_workspace/assessment/test_story_filters.py

```
@@ -45,7 +47,7 @@
 
 class TestStoryFilters(BaseTest):
     base_uri = "/api/assess"
-    _multi_value_filters = {"source", "group", "story_ids", "tags"}
+    _multi_value_filters = {"source", "group", "story_ids", "tags", "language"}
```

Ruff (0.15.13)

[warning] Mutable default value for class attribute

(RUF012)